posts/2023-06-09-setting-up-a-home-energy-monitoring-system.adoc

```diff
@@ -0,0 +1,711 @@
+
+Repeating the process to select the appropriate numbers, we get:
+
+image::Grafana-finding-ratios-3-4-5.png[Finding production ratios - 3-4-5]
+
+The amount of energy produced in each section of the production curve is 4.21 kWh, 12.75 kWh, and 2.97 kWh. When we express this as relative to total production number, we get the following ratios:
+
+* Ramp-up: 21.1%
+* Steady state: 64.0%
+* Ramp-down: 14.9%
+
+The beautiful thing about this is that this exploration and calculation takes just a minute to complete.
+
+=== The "Energy Stats - Consumer details" dashboard
+
+This dashboard shows total energy consumption for all the measurements taken by the _Smappee Infinity_ system (see the _Smappee Infinity_ xref:smappee-infinity-connection-diagram[connection diagram]).
+
+It allows us to answer questions like:
+
+* What was the total energy consumption of:
+** the air-conditioning system last month?
+** the charging of the hybrid car's battery last winter?
+** cooking fresh tomato sauce from the 15kg of tomatoes we bought the other day?
+
+Ok, the last example is a bit far fetched but possible nevertheless and it's a great topic to kickstart a conversation about energy (and cooking)!
+
+The graphs can also tell us some interesting things like:
+
+* How does the power consumption of the air-conditioning system evolve between start-up and the reaching of a steady state?
+* How does the power used to charge the battery of an electrical bicycle evolve during a charging cycle?
+
+For the question, we can have a look at this graph which display the airco power consumption on April 7^th^ of this year:
+
+image::power-profile-airco.png[Power consumption profile for airco]
+
+After an initial spike (with a peak of 2kW) for about 20 minutes, the system evolves to a steady state where it consumes about 0.4kW for about 4 hours, after which changes to a kind of on-off mode.
+
+As for the second question, we can have a look at the following graph with data from May 15^th^:
+
+image::power-profile-electrical-bike-charging.png[Power consumption profile for charging electrical bicycle]
+
+We see that the power at which the battery is charged rises from an initial value of ±160W to 180W over a period of almost 2 hours. I think this is because it wasn't very warm outside, the battery's temperature rises during charging and the charging current is probably a function of that temperature, but that's an unconfirmed and personal theory. After reaching the peak, the charging slows down and stops after another one and a half hour.
+
+In conclusion, we can learn a lot about the different electrical consumers in the house and even learn a few special things about certain devices like the air conditioning.
+
+=== The Power Quality dashboard
+
+This dashboard contains two graphs that display the following data for all channels on the _Smappee Infinity_:
+
+* Apparent and Real power 
+* cos(&#966;)
+
+We can have a look at the data for the garage when the hybrid car is charged:
```

Comment:
```suggestion
We can have a look at the data for the garage when the hybrid car is being charged:
```
Are you referring to the period during which the car is being charged or after it's been charged?